### ISSUE TITLE
Color contrast does not meet minimum requirement - SiteImprove - Search label

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#9460]
# Problem

I believe SiteImprove has issue with a `display: none` span that has the same color as the background (mobile).

# Background

https://my2.siteimprove.com/Inspector/1009549/A11Y/Page?pageId=47390440940&impmd=8BF541F76C11646F59D0764F0BD0CFC8&issue=failed&decision=dismissed+falsePositive+none&comp=ada#/sia-r69/failed/YnYxbmNFajU=;MzgyOGMxMWU=

<img width="812" height="848" alt="Image" src="https://github.com/user-attachments/assets/038f52f9-665f-4197-9e7d-e0e136c8739d" />

**Comment by briand44** [comment]
Still not finding this in the cross site accessibility report when filtering by A and AA conformance. Went through 190 of 288 color contrast issues. Sometime I get frustrated with Siteimprove.

**Comment by joewhitsitt** [comment]
Boom! 367 instances on Provost down to 0 after yesterday's deployment. Not sure how long it will take for the dashboards to stop showing it, but when you click in, it is no longer returning as an issue.

<img width="349" height="370" alt="Image" src="https://github.com/user-attachments/assets/8829f116-01c8-41c8-97df-0975fff19bc7" />

**Comment by pyrello** [comment]
I'm going to be looking into this issue next as part of #9587. Re-opening until we get to the bottom of it.
